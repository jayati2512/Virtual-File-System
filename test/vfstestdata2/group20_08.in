createvfs demo8 1000
mountvfs demo8
removefile /home/abc.h
addfile /home abc.h /usr/include/stdio.h
makedir / home
addfile /home abc.h /usr/include/stdio.h
deletedir /home/abc.h
movedir /home/abc.h /
addfile /home abc.h /usr/include/stdio.h
updatefile /home/abc.h /usr/include/memory.h
copyfile /home/abc.h /abc.h
listfile /home/abc.h /tmp/abc.h
removefile /home/abc.h
exportfile /home/abc.h
unmountvfs demo8
mountvfs demo8
makedir /home/a/b/c d
movefile /abc.h /home/a/b/c/d/abc.h
copyfile /home/a/b/c/d/abc.h /abc.h
unmountvfs demo8
