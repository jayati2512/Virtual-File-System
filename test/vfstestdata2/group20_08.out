createvfs_SUCCESS
mountvfs_SUCCESS
removefile_FAILURE CANNOT_FIND_SPECIFIED_FILE
addfile_FAILURE PATH_NOT_FOUND
makedir_SUCCESS
addfile_SUCCESS
deletedir_FAILURE NOT_A_DIR
movedir_FAILURE SOURCE_CANNOT_BE_FILE
addfile_FAILURE FILE_ALREADY_EXISTS
updatefile_SUCCESS
copyfile_SUCCESS
listfile_FAILURE NOT_A_TEXT_FILE
removefile_SUCCESS
exportfile_FAILURE VFS_INSUFFICIENT_ARGUMENTS
unmountvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
movefile_SUCCESS
copyfile_SUCCESS
unmountvfs_SUCCESS
